server/service/software_installers.go

```diff
@@ -262,6 +399,13 @@ func (svc *Service) GetSoftwareInstallerMetadata(ctx context.Context, _ bool, _
 	return nil, fleet.ErrMissingLicense
 }
 
+type getSoftwareInstallerResponse struct {
```

Comment:
Shouldn't this be `updateSoftwareInstallerResponse`?

Reply:
It's the result of a `PATCH`, which should just be the modified entity, and is an extract of the `getSoftwareTitleResponse` specific to the installer. We don't yet have an endpoint that just GETs installer metadata, but when we do this will be the format returned.

If you feel strongly that we rename this we can, but I think this is semantically correct now, and will be more correct when we need an endpoint that's just a GET (which I think we'll want once we start supporting multiple installers per title X team).

Reply:
Resolving this so I can bounce through the remaining feedback. Feel free to bring this back in a final round of review.